src/io/CavityWriter.h

```diff
@@ -5,17 +5,34 @@
 
 #include "plugins/PluginBase.h"
 
+class CavityEnsemble;
+
 class CavityWriter : public PluginBase {
```

Comment:
documentation of class is missing

Reply:
Changed as requested c9a26ad90439b66e658c293bae6cb914eba4b29c